loopback.cc

```diff
@@ -35,10 +34,18 @@ int loopback_init(const char* device, int w, int h, int debug) {
 	int ret_code = 0;
 
 	fdwr = open(device, O_RDWR);
-	assert(fdwr >= 0);
+	if(fdwr < 0) {
+		perror(__func__);
```

Comment:
We might need to switch to the more verbose `fprintf(stderr, "%s:%d:%s\n", __FILE__, __LINE__, strerror(errno));` format to include line numbers - removing ambiguity about which error has occurred in a function? Obviously if a function only has one error condition, we're good!

Reply:
I tried to avoid this ambiguity with a separate error description just below. But updating it aas suggested to use `__FILE__`, `__LINE__` and `__FUNCTION__` shouldn't be an issue.